rust/decentraland-godot-lib/src/dcl/js/js_modules/Players.js

```diff
@@ -1,3 +1,20 @@
-module.exports.getPlayerData = async function (body) { return {} }
-module.exports.getPlayersInScene = async function (body) { return {} }
-module.exports.getConnectedPlayers = async function (body) { return {} }
\ No newline at end of file
+module.exports.getPlayerData = async function (body) {
+    let res = await Deno.core.ops.op_get_player_data(body.userId);
```

Comment:
```suggestion
    const res = await Deno.core.ops.op_get_player_data(body.userId);
```

It's better JavaScript Code.
It applies to the below `let res` too